MediaBrowser.MediaEncoding/Probing/MediaStreamInfo.cs

```diff
@@ -310,5 +310,12 @@ public class MediaStreamInfo
         /// <value>The color primaries.</value>
         [JsonPropertyName("color_primaries")]
         public string ColorPrimaries { get; set; }
+
+        /// <summary>
+        /// Gets or sets the side_data_list.
+        /// </summary>
+        /// <value>The side_data_list.</value>
+        [JsonPropertyName("side_data_list")]
+        public IReadOnlyList<IReadOnlyDictionary<string, string>> SideDataList { get; set; }
```

Comment:
Why a dictionary? You could create a new class

```C#
public class MediaStreamSideData
{
    [JsonPropertyName("dv_profile")]
    public int? DvProfile { get; set; }

    ...
}
```

Reply:
Done.